spark-bigquery-dsv2/spark-bigquery-dsv2-common/src/main/java/com/google/cloud/spark/bigquery/v2/context/BigQueryDataSourceWriterModule.java

```diff
@@ -103,35 +104,39 @@ public BigQueryIndirectDataSourceWriterContext provideIndirectDataSourceWriterCo
 
   Path createGcsPath(SparkBigQueryConfig config, Configuration conf, String applicationId)
```

Comment:
Method should removed in favor of hte `SparkBigQueryUtil` one

Reply:
Ah good catch